samples/web/content/apprtc/appwindow.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
I think this will be the next thing that I do (the build/packaging steps), in a separate CL.